File: src/com/Torvald/ColourUtil/HSV.java
package com.Torvald.ColourUtil;

/**
 * Created by minjaesong on 16-03-10.
 */
public class HSV {
    private float H;
    private float S;
    private float V;

    public HSV() {
    }

    /**
     *
     * @param h 0-359
     * @param s 0-1
     * @param v 0-1
     */
    public HSV(float h, float s, float v) {
        H = h;
        S = s;
        V = v;
    }

    public float getH() {
        return H;
    }

    public void setH(float h) {
        H = h;
    }

    public float getS() {
        return S;
    }

    public void setS(float s) {
        S = s;
    }

    public float getV() {
        return V;
    }

    public void setV(float v) {
        V = v;
    }
}

File: src/net/torvald/colourutil/HSV.kt
package net.torvald.colourutil

/**
 * Created by minjaesong on 16-03-10.
 */
/**
 * @param h : Hue 0-359
 * @param s : Saturation 0-1
 * @param v : Value (brightness in Adobe Photoshop(TM)) 0-1
 */
data class HSV(
    var h: Float,
    var s: Float,
    var v: Float
)
